app/controllers/spree/checkout_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Spree
+  class CheckoutController < Spree::StoreController
```

Comment:
I think it's worth adding the commit message here as doc for the next dev. Perhaps someone else finds out in the future. Also, the purpose of this controller will become more obvious.

Reply:
good point! done.